scipy/stats/_crosstab.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+
+
+def crosstab(*args, levels=None):
```

Comment:
No firm opinion on my end but did you consider a `**kwargs` instead of individual arguments after the args tuple unpacking? Just thinking that `**kwargs` might make the signature more extensible in future.

Reply:
spoke w/Ralf about this one, I was thinking the use of `**kwargs` would make the signature more flexible but the price of that flexibility wouldn't really give any benefit and we'd lose the a bunch so the way the signature is here is fine with me.